### ISSUE TITLE
Odoo web throws 404 when requesting static assets without session cookie breaking webcache

### ISSUE THREAD

**Issue by gvansanden** [odoo/odoo#53766]
Impacted versions: 13.0

Steps to reproduce:
Load the /web page in a browser, copy/paste a link to a css file to curl

Current behavior:
The request gets a 404 response

Expected behavior:
Static assets like images and CSS should load without the cookie and a 404 is an incorrect response anyway.

Video/Screenshot link (optional):
curl https://odoo-test.diamantipertutti.com/web/content/138897-e593ca4/1/web.assets_common.css
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again.</p>

Not only does this behavior break with the normal use of http status code, it breaks caching proxies like varnish that should be caching static content.

**Comment by Yenthe666** [comment]
@gvansanden you might want to hide your database manager buddy, that's a possible security issue. 😉

**Comment by gvansanden** [comment]
@Yenthe666 yes, we have an issue that prevents us doing that on test currently
